.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: release
+on: workflow_dispatch
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Verify branch name
+        if: github.ref != 'refs/heads/master'
+        run: |
+          echo "🚨 The release must start from the master branch!"
+          exit 1
+
+      - name: Configure releaser details
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+
+      - name: Install pnpm
+        uses: wyvox/action-setup-pnpm@v3
+        with:
+          pnpm-version: 8.5.1
+          node-version: 18.x
+          node-registry-url: "https://registry.npmjs.org"
+
+      - name: Trigger release script
+        run: pnpm run release
```

Comment:
do we need the 'run' keyword?

Reply:
Indeed non necessarily, but I’d like to keep it for consistency with other packages 🙂